code/src/nuvla/job/job.py

```diff
@@ -265,6 +265,12 @@ def get_user_api(self):
                                 f'{authn_info["active-claim"]} '
                                 f'{" ".join(authn_info["claims"])}')
 
+    @property
+    def target_resource(self) -> str:
+        return self['target-resource']['href']
+
+    nuvlaedge_id = target_resource
```

Comment:
The target-resource is not always a NuvlaEdge so we cannot say `nuvlaedge_id = target_resource`

Reply:
ACK